src/pages/student/categories/CategoryList/index.js

```diff
@@ -37,28 +37,31 @@ function CategoryList() {
     return categories.map((category, idx) => {
       return (
         <Card className={style.card} key={idx}>
-          <Card.Header id={style.cardHeader}></Card.Header>
+          <Card.Header id={style.cardHeader}>
+            <p className={style.cardTitle}>{category.name}</p>
+            <p className={style.cardSubtitle}>{category.description}</p>
+          </Card.Header>
           <Card.Body>
             {category.subcategories_count ? (
               <Link to={`/categories/${category.id}/sub`}>
-                <div className={style.cardContent}>
-                  <center>
-                    {' '}
-                    <Card.Title className={style.cardTitle}>
-                      {category.name}
-                    </Card.Title>{' '}
-                  </center>
+                <div>
+                  {' '}
+                  <Card.Title >
+                    <a id={style.Subtitle}>
```

Comment:
- [ ] Change <a> tag to <p> instead to avoid browser errors shown in console.
![Screenshot from 2021-11-22 11-40-38](https://user-images.githubusercontent.com/88467532/142797534-1631e5ec-ccd1-47ad-b1b5-816a2c71b77f.png)